treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -197,8 +198,14 @@ def save_local_data(self):
             print('Saving {0} data...'.format(table_name))
             model_values = properties['model'].objects.using(self.source).filter(
                 pk__in=self.models_instances[table_name])
+            self._ignore_sensitive_fields(table_name, model_values)
             properties['model'].objects.using(self.target).bulk_create(model_values)
 
+    def _ignore_sensitive_fields(self, table_name, model_values):
```

Comment:
Could you please add a note for the function as to why we need to ignore some fields?

Reply:
Sure thing.